apps/wallet-dashboard/components/staked-timelock-object/StakedTimelockObject.tsx

```diff
@@ -52,14 +50,18 @@ export function StakedTimelockObject({
     const [sumPrincipalFormatted, sumPrincipalSymbol] = useFormatCoin({
         balance: summary.principal,
     });
-
     return (
         <Card onClick={() => handleUnstake(timelockedStakedObject)}>
             <CardImage>
-                <ImageIcon src={null} label={name} fallback={name} size={ImageIconSize.Large} />
+                <ImageIcon
+                    src={validatorMeta?.imageUrl || null}
+                    label={validatorMeta?.name || ''}
+                    fallback={validatorMeta?.name || ''}
```

Comment:
label and fallback use to fallback to `validatorAddress` if there is validatorMeta?.name, should we add this back in?